cmd/ltops/db.go

```diff
@@ -42,11 +42,14 @@ func shell(cluster *ltops.Cluster) error {
 
 	logrus.Infof("Connecting to %s:%d", settings.Endpoint, settings.Port)
 	var cmd *exec.Cmd
-	if (*cluster).Configuration().DBEngineType != "aurora-postgresql" {
+
+	switch (*cluster).Configuration().DBEngineType {
+	case "aurora-postgresql":
```

Comment:
Oops, I think you flipped this? It would be super clear to explicitly list out `aurora`, `aurora-mysql` and `aurora-postgresql`.

Reply:
🤦‍♂